docs/paas+/dev/onboarding-to-paas+.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Onboarding to PaaS+
+description: Get your project ready and start upgrading with PaaS+
+template: concept-topic-template
+---
+
+Onboarding to PaaS+ happens in two steps:
+
+1. Getting your project ready and compliant with a set of pre-requirements to ensure you fully benefit from all the features.
```

Comment:
`1.3 Make your code compliant with Spryker Quality Gate` is part of `1. Get your project ready for PaaS+`. Where is the new separate step coming from?

Reply:
Maybe we don't need to further divide right now. 
The reason to separate "assessment" from "make the project ready" is because moving forward we may have different owners for the steps (e.g. maybe Spryker does the assessment and the customer makes the project compliant). 
But let's keep it simple for now. You can leave it as it is. Thanks!